### PR TITLE
UserID branch

### PR DESCRIPTION
**Pull request by AbdelSallam**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -7,3 +7,4 @@ config.yaml
 example.db
 sendToServer.sh
 tweets-*.json
+/archived tweetsdb
```
Comment (jchavannes) [inline .gitignore:10]: I don't know that I've seen a space in `gitignore` before. Is this a file with a space in the name?

File: db/address_key.go [hunk 2]
```diff
@@ -3,21 +3,22 @@ package db
...
+	"strconv"
 	"strings"
 )
 
 type AddressLinkedKey struct {
-	Address     string
-	TwitterName string
-	Key         []byte
+	Address string
+	UserID  string
```
Comment (jchavannes) [inline db/address_key.go:12]: Why not store UserId as an `int64`?

File: db/flag.go [hunk 3]
```diff
@@ -21,17 +22,17 @@ func GetDefaultFlags() Flags {
 }
 
 type Flag struct {
-	Address     string
-	TwitterName string
-	Flags       Flags
+	Address string
+	UserID  string
```
Comment (jchavannes) [inline db/flag.go:26]: Same as `AddressLInkedKey`, why not `int64`? Also the rest of db items that use `UserId`

**Review by jchavannes — COMMENTED** [review]
Also a couple `println`s.

File: .gitignore (hunk 1 above)
Comment (AbdelSallam) [reply .gitignore:10]: Yeah it is. I can just delete it though, it was for testing the migrate command

**Pull request merged**